### ISSUE TITLE
AMP accordion and accessibility on Lighthouse audit

### ISSUE THREAD

**Issue by AndrewKGuan** [ampproject/amphtml#18516]
amp-accordion is failing Lighthouse's accessibility test. The message from the test is the following:

<id> attributes on the page are not unique 
The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies. 

How to reproduce:
run Lighthouse audit against any AMP that has an accordion.
